codegen/src/main/scala/akka/grpc/gen/scaladsl/ScalaBothCodeGenerator.scala

```diff
@@ -4,6 +4,6 @@
 
 package akka.grpc.gen.scaladsl
 
-object ScalaBothCodeGenerator extends ScalaServerCodeGenerator with ScalaClientCodeGenerator {
+object ScalaBothCodeGenerator extends ScalaServerCodeGenerator(powerApis = false) with ScalaClientCodeGenerator {
```

Comment:
I actually implemented this change and then was reminded when I tried to use the updated code on a project that we can't pass through to scalaPB our additional code generator params without also modifying it to handle them -- otherwise it throws an "unrecognized parameter" exception. That's why I didn't do it that way originally.